design-patterns/gc-institution/gc-institution-fr.html

```diff
@@ -0,0 +1,50 @@
+---
+{
+	"title": "À propos de l'institution",
+	"description": "Exemple pratique pour la configuration de conception « À propos de l'institution »",
+	"language": "fr",
+	"altLangPage": "gc-institution-en.html",
+	"breadcrumbs": [
+		{ "title": "À propos de l'institution", "link": "design-patterns/gc-institution/gc-institution-doc-fr.html" }
+	],
+	"dateModified": "2024-03-07",
+	"pageclass": "wb-prettify all-pre"
+}
+---
+
+<p class="mrgn-bttm-lg">Vous trouverez ci-dessous un exemple pratique pour la configuration de conception « À propos de l'institution ».</p>
+
+<section class="row">
+	<div class="col-md-8 bold-content">
```

Comment:
Why the `col-md-8`? That use seem to be a workaround of the limited width content feature.  We should use the limited width content feature instead.

A grid can be useful when there is multiple column, not just one which is use not for the column purpose but for limiting the content width.

May be we will need to extend the content limited width when there is a colcount of 2 feature.